src/main/java/com/grupozap/dumping_machine/config/ApplicationProperties.java

```diff
@@ -74,7 +74,7 @@ public String getPartitionPattern(){
         if(partitionPattern != null){
             return partitionPattern;
         } else {
-            return "'dt='yyyy-MM-dd'/hr='HH'/'";
+            return "'dt='yyyy-MM-dd'/hr='HH";
```

Comment:
Does it need to remove the quote?

Reply:
Yes, just to avoid create empty partitions. I just moved the `/` to other point. If i keep the `/`, it creates an empty partition.

Reply:
Got it :nerd_face: